gordo_components/cli/cli.py

```diff
@@ -386,11 +352,70 @@ def get_all_score_strings(metadata):
     return all_scores
 
 
+def validate_ip(ctx, parameter, value):
```

Comment:
Maybe better to follow what we've been doing with custom click params in cli.custom_types Whether you want to create a object out of it, or at least move these function into that.

Reply:
Good to know :ok_hand: